README.md

```diff
@@ -60,7 +60,7 @@ in the [contrib][contrib] folder:
 
 Installation
 ------------
-mistletoe is tested for Python 3.3 and above. Install mistletoe with pip:
+mistletoe is tested for Python 3.4 and above. Install mistletoe with pip:
```

Comment:
OK, looking at <https://w3techs.com/technologies/details/pl-python/3>, I hope their data are correct, v3.3 is only at 0,1%, v3.4 at 2,6%, v3.5 at 8,4% and the mostly used v3.6 at 50%. So we could possibly also drop support for v3.4 as discussed at #134 [here](https://github.com/miyuchina/mistletoe/pull/134#discussion_r805237939) where we talk about requiring v3.5 for `typing`. What do you think?

Reply:
Also, we shouldn't forget about removing versions from `.travis.yml` as mentioned at #121.

Reply:
As far as I am concerned such usage statistics don't matter. What matters is that Python < 3.6 is no longer supported by the Python Software Foundation, see https://endoflife.date/python.

Ok I bumped the version to 3.5 & updated the `.travis.yml`.

Sidenote: The Travis CI does not actually appear to be running? https://app.travis-ci.com/miyuchina/mistletoe

Reply:
OK, thanks.

> Sidenote: The Travis CI does not actually appear to be running? https://app.travis-ci.com/miyuchina/mistletoe

Yes, AFAIK this project is not on the new `.com` instance of Travis yet. Discussion is welcome at #121. :)